Dockerfile.ci

```diff
@@ -1016,46 +974,6 @@ function check_airflow_python_client_installation() {
     python "${IN_CONTAINER_DIR}/install_airflow_python_client.py"
 }
 
-function start_webserver_with_examples(){
```

Comment:
Shouldn't this start using the api server, instead of being removed completely?

Reply:
Ah yes. ... Now I know why the pytest client failed... that was a feature specifically added to **actually** treat `test_python_client.py` as a test to run with `pytest` ... Now I remember it ! 🤦